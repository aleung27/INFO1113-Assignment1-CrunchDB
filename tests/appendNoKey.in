APPEND y 70
SET k 7
APPEND K 241
APPEND k 1 2 3
LIST ENTRIES
