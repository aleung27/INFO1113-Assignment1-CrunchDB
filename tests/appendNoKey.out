> no such key

> ok

> no such key

> ok

> k [7 1 2 3]

